### ISSUE TITLE
Current Main Navigation Not Highlighted

### ISSUE THREAD

**Issue by yatil** [w3c/wai-website-theme#122]
## Expected behavior:

1. Go to https://www.w3.org/WAI/planning/statements/minimal-example/
2. At the main navigation:
  * The current main navigation item (Planning & Policies) should have `aria-current` set to `location`
  * The current main navigation item (Planning & Policies) should be highlighted (which is triggered by `aria-current="location"`)

## Current behavior:

`aria-current="location"` is not set.

(Edit to change `aria-current="true"` to `aria-current="location"`. The editors regret the error.)
